gdk4-x11/Gir.toml

```diff
@@ -89,6 +89,19 @@ name = "Gdk.Surface"
 status = "manual"
 final_type = false
 
+[[object]]
+name = "GdkX11.*"
+status = "generate"
+    [[object.function]]
+    name = "x11_free_compound_text"
+    ignore = true
+    [[object.function]]
+    name = "x11_free_text_list"
+    ignore = true
+    [[object.function]]
+    name = "x11_lookup_xdisplay"
+    ignore = true # TODO: manually implementation needed
```

Comment:
Why? That looks like the other functions to me :)

Reply:
it takes a xlib::Display, which doesn't implement the glib traits. so you can't do xdisplay.to_glib_none ;-)

Reply:
I see